internal/transactions/sign.go

```diff
@@ -105,10 +105,15 @@ func sign(
 	})
 
 	for _, signer := range signers {
-		signed, err = services.Transactions.Sign(signer, payload, globalFlags.Yes)
+		if !globalFlags.Yes && !output.ApproveTransactionForSigningPrompt(tx) {
```

Comment:
What is globalFlags.Yes ?

Reply:
:D there's a `-Y` global flag that allows you to accept any prompts, that's an established pattern in CLIs.